radixdlt/src/integration/java/com/radixdlt/integration/distributed/simulation/network/SimulationNetwork.java

```diff
@@ -128,7 +128,7 @@ public String toString() {
 	@Inject
 	public SimulationNetwork(ChannelCommunication channelCommunication) {
 		this.channelCommunication = Objects.requireNonNull(channelCommunication);
-		this.receivedMessages = ReplaySubject.<MessageInTransit>createWithSize(20) // To catch startup timing issues
+		this.receivedMessages = ReplaySubject.<MessageInTransit>createWithSize(128) // To catch startup timing issues
```

Comment:
Maybe this should be a function of the number of nodes in the system?

Also, nice catch.  Must have been a bit of a pain to track down.